governance.md

```diff
@@ -24,17 +28,57 @@ Every code owner is expected to:
 * Approve only those PRs in which the contributor made the requested improvements
 * Check if the related CI tests have completed successfully before approving the PR
 * Make sure that the PR approval flow runs smoothly
-* Prioritize issues and manage their processing
 * Proactively fix bugs
 * Perform maintenance tasks for their projects
-
-### Add or remove a code owner
+* Issue that is assigned to current release is treated with priority by maintainer that is an assignee
+* Monitor [Slack](http://slack.kyma-project.io)
+* Triage GitHub issues and perform pull request reviews for other maintainers and the community. Follow [labels workflow](/governance/#issues-workflow-issues-workflow) document
+* In general continue to be willing to spend at least 50% of one's time working on Kyma (~2.5 business days per week).
+
+### Process for becoming a maintainer aka codeowner
+
+* Express interest to the existing maintainers that you or your organization are interested in becoming a
+  maintainer. The difference is that in case you are individual or corporate contributor/maintainer then different contributor license agreement applies.
+* Express what are the areas of Kyma that you want to engage with. Kyma, because of its goal, is complex and it is impossible to become a maintainer in every area. Make sure you get support from proper group of maintainers.
+* Becoming a maintainer generally means that you are going to be spending substantial time (>50%) on Kyma for the foreseeable future. Among other responsibilities this is:
+  * creating and merging pull-request
+  * reviewing pull-request
+  * creating or commenting on issues
+  * helping on Slack and other community communication channels
+* We will expect you to start contributing increasingly complicated PRs, under the guidance
+  of the existing maintainers.
+* We may ask you to do some PRs from our backlog.
+* As you gain experience with the code base and our standards, we will ask you to do code reviews for incoming PRs (i.e., all maintainers are expected to shoulder a proportional share of community reviews).
+* After a period of approximately 3 months of working together and making sure we see eye to eye, candidate for maintainer should create an issue in [community](https://github.com/kyma-project/community) repository with list of his current contributions. The existing maintainers will confer and decide whether to grant maintainer status or not. We make no guarantees on the length of time this will take, but no longer than 3 months.
```

Comment:
nit: maybe as a part of that PR we can add a new issue template for becoming the maintainer?

here is how it looks like in k8s: https://github.com/kubernetes/org/blob/master/.github/ISSUE_TEMPLATE/membership.md

Reply:
you think it makes sense really to do it in this PR? maybe let us wait for the first issue of someone who wants to become a maintainer? 

just to be clear, I love github issues templates, just do not want to complicate this PR, and adding issue later will be much easier, a kind of formality and only approval from technical writers